### PR TITLE
Create comment-issue.yml

### PR DESCRIPTION
**Pull request by jankapunkt**
## What

Proposal for automated comments of newly created issues

## Why

Let contributors know they can also open a PR for trivial issues and that sponsoring is important :-D

## Checklist

- [ ] Agreed upon solution from issue #
- [ ] Stakeholders have approved the changes

### CODE REVIEW

**Comment by jankapunkt** [comment]
@StorytellerCZ if you think this is okay I would add this to all our repositories

File: .github/workflows/comment-issue.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          issue-number: ${{ github.event.issue.number }}
+          body: |
+            Thank you for submitting this issue!
+            
+            We, the Members of Meteor Community Packages take every issue seriously.
+            Our goal is to provide long-term lifecycles for packages and keep up
+            with the newest changes in Meteor and the overall NodeJs/JavaScript ecosystem.
+            
+            However, we contribute to these packages mostly in our free time.
+            THerefore, we can't guarantee you issues to be solved within certain time.
```
Comment (StorytellerCZ) [inline .github/workflows/comment-issue.yml]: Typo here.

**Review by StorytellerCZ — CHANGES_REQUESTED** [review]

**Comment by StorytellerCZ** [comment]
Great idea! Definitely worthy addition.

**Comment by jankapunkt** [comment]
Great, I will add this to the other repos now

**Pull request merged**